politeiawww/cmd/pictl/cmdproposaledit.go

```diff
@@ -157,17 +162,63 @@ func proposalEdit(c *cmdProposalEdit) (*rcv1.Record, error) {
 			return nil, err
 		}
 		c.Name = pm.Name
-	case c.Random && c.Name == "":
```

Comment:
It would not hurt to document the case statements since they are very hard to interpret.